### DISCUSSION TITLE
Set Stack Directory, on "This stack is not managed by Dockge"

### DISCUSSION THREAD

**Discussion by tholum** [louislam/dockge#679] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

API, UI Feature

### 🔖 Feature description

For my development server, I have a few different docker compose folders, the main /opt/stack, but I also have /home/<username>/projects/.... for a few different projects, it would be great if I could set the directory for the stack, and have it manage it on the one's that are "This stack is not managed by dockge"

### ✔️ Solution

(Assuming the Folder is mapped in volumes in the dockge container )

Next to the "This stack is not managed by Dockge", create a button "Set Path"
![image](https://github.com/user-attachments/assets/ea031341-9aa7-48cc-a08d-067a3fe5ca8f)

When this is clicked, it brings up a dialog and lets you set the path, This stores the location for this folder, and allows it to be managed

### ❓ Alternatives

Creating a second Dockge instance, and connect it as an agent, but when I do this, it duplicates everything ( Seeing they are running on the same docker backend )

**Comment by voltagex** [comment]
https://github.com/louislam/dockge/blob/01906205f019e6a215551f644ec732c7c8e6086b/backend/stack.ts#L106 seems very strict, but I haven't looked at the flow on effects from changing this
